inst/cloudml/setup.py

```diff
@@ -34,7 +34,7 @@
     # ml-engine doesn't provide TensorFlow 1.3 yet but they could be potentially
     # upgraded; however, we've found out some components (e.g. tfestimators) hang even
     # under python when upgrading TensorFlow versions.
-    # ["pip", "install", "keras", "--upgrade"],
+    ["pip", "install", "keras", "--upgrade"]
```

Comment:
We need some additional dependencies to get all of the keras feature-set (see https://tensorflow.rstudio.com/tools/installation.html#additional-dependencies)

```bash
pip install h5py pyyaml requests Pillow scipy
```

Reply:
ah, great, added.